### ISSUE TITLE
The recon mission

### ISSUE THREAD

**Issue by rekterakathom** [rekterakathom/Overthrow#35]
The recon mission cannot be completed due to the enemies never being marked as spotted.
This is a prio problem to fix because this is hands down the most asked about thing in the Overthrow discord

**Comment by Tuupertunut** [comment]
I cannot reproduce this issue. I tried to recon Swarog Base in Livonia, there were 9 soldiers, 1 vehicle and 1 machine gun. The mission was completed after I spotted these all.

**Comment by Dzemoglowy** [comment]
> Idea: A recon mission should complete after spotting around 80% of the troops stationed there, instead of 100%. Now even if someone is inside a house, the mission could still be completed.

I agree with that. I done recon mission several times on fuel depot near Laikoro and clue is to wait when all patrol will arrive on player's sight.

**Comment by PierreO80** [comment]
The problem with this mission appear only when mission hosted on a Dedicated Server. In a local hosted it works...

**Comment by rekterakathom** [comment]
> The problem with this mission appear only when mission hosted on a Dedicated Server. In a local hosted it works...

Looking at the code, I can't see any reason for this to be the case. I didn't do any testing though, it could be that there is a bug in the engine / documentation, so BI fault. I'll see what I can do to make this mission complete more reliably.

**Comment by PierreO80** [comment]
Hello, thanks for your reply. I don't know the reason but it seems this missions have some scripts broken on a DS host. For example, you cannot delete things (like safe) or vehicles with ACE interaction like on local host, and another example is the fact that on DS you can open your inventory and put drugs away during nato searching inspection (not the case in local host we can't open the inventory during those inspection).
